cms/envs/common.py

```diff
@@ -84,6 +84,9 @@
     # django-debug-toolbar
     DEBUG_TOOLBAR_PATCH_SETTINGS,
     BLOCK_STRUCTURES_SETTINGS,
+
+    # Default currency, DOES NOT AFFECT VERIFIED STUDENTS
```

Comment:
@rlucioni CourseMode.modes_for_course is called a number of times in the CMS. This function needs the currency setting for the default course mode if none exists for that course.